molgenis-ui-settings/src/main/java/org/molgenis/settings/controller/SettingsController.java

```diff
@@ -0,0 +1,37 @@
+package org.molgenis.settings.controller;
+
+import org.molgenis.data.i18n.LanguageService;
+import org.molgenis.data.settings.AppSettings;
+import org.molgenis.security.user.UserAccountService;
+import org.molgenis.ui.controller.VuePluginController;
+import org.molgenis.ui.menu.MenuReaderService;
+import org.molgenis.web.PluginController;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import static org.molgenis.settings.controller.SettingsController.URI;
+
+@Controller
+@RequestMapping(URI)
+public class SettingsController extends VuePluginController
```

Comment:
I know it's a minimal controller , but maybe setup a test for it anyway, this makes is easier to add/update test when functionality is added/updated.